app/data/census_household.json

```diff
@@ -1429,12 +1429,17 @@
                                                 {
                                                     "label": "Yes, an address outside the UK (please specify)",
                                                     "value": "Other",
-                                                    "other": {
-                                                        "label": "Please enter country"
-                                                    }
+                                                    "child_answer_id": "another-address-answer-other"
```

Comment:
Having to define a two way relationship seems like a configuration overhead with room for error. At the very least we'd need some validation to check this but going back to my point it seems like a configuration overhead if we know the mapping already